.github/workflows/cypress.yaml

```diff
@@ -43,30 +45,35 @@ jobs:
     - uses: actions/checkout@v4
     - name: Install Edge
       uses: browser-actions/setup-edge@v1
-      if: ${{ matrix.cypress-browser == 'edge' }}
+      if: ${{ matrix.cypress-browser == 'edge' && github.event_name != 'pull_request' }}
     - name: Set up system
       run: bin/before_install
+      if: ${{ matrix.cypress-browser == 'chrome' || (matrix.cypress-browser != 'chrome' && github.event_name != 'pull_request') }}
```

Comment:
I know this is WIP, so future cleanup idea.

I'm thinking let's shove this either into an ENV var with a sensible name, then use that later.  So something like

```yaml
    env:
      TEST_SUITE: spec:cypress
      RUN_CYPRESS: ${{ matrix.cypress-browser == 'chrome' || (matrix.cypress-browser != 'chrome' && github.event_name != 'pull_request') }}
      ...
    steps:
    ...
    - name: Set up system
      run: bin/before_install
      if: ${{ env.RUN_CYPRESS }}
```

Reply:
Though I think env might be cleaner and I think it can support booleans

Reply:
I like the name.  I think we might want pushes, manual / adhoc, and scheduled runs to be full browser support.  Either way, we can tweak it once we have better ideas.